node/clients/checkout.ts

```diff
@@ -0,0 +1,215 @@
+import { AuthenticationError, ForbiddenError, InstanceOptions, IOContext, JanusClient, RequestConfig, UserInputError } from '@vtex/api'
+import { AxiosError } from 'axios'
+
+const statusToError = (e: any) => {
```

Comment:
We should make this into an util because lots of clients are using it. cc @tlgimenes @salesfelipe

Reply:
sales did it in his PR, I believe, we can merge it later